src/components/common/FaqItems.tsx

```diff
@@ -29,16 +33,16 @@ const DEFAULT_STATE: IFaqItemsState = {
   panel: "",
 };
 
-const getFaqContent = (lang: "de" | "en"): Array<IFaqContent> => {
-  return translated(lang).faqs.content;
-};
-
-class FaqItems extends React.Component<IFaqItemProps, IFaqItemsState> {
-  constructor(props: IFaqItemProps) {
+class FaqItems extends React.Component<WithTranslation, IFaqItemsState> {
+  constructor(props: WithTranslation) {
     super(props);
     this.state = DEFAULT_STATE;
   }
 
+  getFaqContent = (): Array<IFaqContent> => {
+    return CONTENT[this.props.i18n.language].faqs.content;
```

Comment:
this loading of faq content we have to change as soon as the data lives in it's own `faqs.json`